### PR TITLE
Adds new 'Solution Explorer' description section.

### PR DESCRIPTION
**Pull request by briandooley**
Also, adds list of applications included in RHMI with a brief description of each.

### CODE REVIEW

**Comment by briandooley** [comment]
@finp Can you review?

File: docs/content/intro-con.adoc [hunk 1]
```diff
@@ -1,25 +1,38 @@
...
 
-
 Red Hat Managed Integration delivers managed integration services hosted on the leading container platform, enabling diverse developers to engage in agile, microservices-based application development.
 {PRODUCT_SHORT} enables your team to develop, build and deploy enterprise integration solutions on an OpenShift Dedicated cluster managed by Red Hat.
 {PRODUCT_SHORT} enables developers adopt event-based computing and API-based development with lightweight, loosely coupled, self-service and flexible components.
 {PRODUCT_SHORT} enables the operations team focus on high value objectives while Red Hat manage the infrastructure. All the applications and services are protected by Red Hat Single Sign-On.
 
-The following middleware products are available as part of {PRODUCT_SHORT}:
+== Solution Explorer
+The Solution Explorer is a web application that serves as a user friendly gateway to an {PRODUCT_SHORT} cluster.  It contains links to middleware products that are already installed and ready to use with no configuration necessary, with access to each product managed through RHSSO.
```
Comment (pwright) [inline docs/content/intro-con.adoc]: user-friendly gateway... seems colloquial
contains links - might translate badly
product might be misleading

File: docs/content/intro-con.adoc [hunk 2]
```diff
@@ -1,25 +1,38 @@
...
+
+* *Red Hat Developer Launcher*, a platform for continuous application delivery, with a set of templates.
+
+* *Red Hat Fuse Online*, a facility to obtain data from an application or service, operate on that data, and then send the data to another application or service without writing code.
 
+* *Red Hat Single Sign-On*, a single sign-on solution for web apps and RESTful web services.
+
+* *Red Hat 3scale API Management*, a platform that provide RESTful API Management.
+
+NOTE: You can log into 3scale directly from the Solution Explorer, however note that:
```
Comment (pwright) [inline docs/content/intro-con.adoc]: needs to be a note block (and rewritten)

File: docs/content/intro-con.adoc (hunk 1 above)
Comment (briandooley) [reply docs/content/intro-con.adoc]: Fixed after conversation with Paul.

File: docs/content/intro-con.adoc (hunk 2 above)
Comment (briandooley) [reply docs/content/intro-con.adoc]: Fixed after conversation with Paul.

**Comment by briandooley** [comment]
@finp ok to merge?

**Review by pwright — APPROVED** [review]

**Pull request merged**